server/issue.go

```diff
@@ -193,6 +193,75 @@ func httpAPIGetCreateIssueMetadata(ji Instance, w http.ResponseWriter, r *http.R
 	return http.StatusOK, nil
 }
 
+func (p *Plugin) assignJiraIssue(mmUserId, issueKey, assignee string) (string, error) {
+	ji, err := p.LoadCurrentJIRAInstance()
+	if err != nil {
+		return "", err
+	}
+
+	jiraUser, err := ji.GetPlugin().LoadJIRAUser(ji, mmUserId)
+	if err != nil {
+		return "", err
+	}
+
+	jiraClient, err := ji.GetJIRAClient(jiraUser)
+	if err != nil {
+		return "", err
+	}
+
+	// check for valid issue key
+	_, _, err = jiraClient.Issue.Get(issueKey, nil)
+	if err != nil {
+		errorMsg := fmt.Sprintf("We couldn't find the issue key `%s`.  Please confirm the issue key and try again.", issueKey)
+		return errorMsg, nil
+	}
+
+	// Get list of assignable assignees
+	url := fmt.Sprintf("rest/api/3/user/assignable/search?issueKey=%s&query=%s", issueKey, assignee)
+	req, err := jiraClient.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+
+	res, err := jiraClient.Do(req, nil)
+	if err != nil {
+		if res.Response.StatusCode == 401 {
+			return "You do not have the appropriate permissions to perform this action. Please contact your Jira administrator.", nil
+		}
+		return "", err
+	}
+
+	var data []*jira.User
+	defer res.Body.Close()
+	if err := json.NewDecoder(res.Body).Decode(&data); err != nil {
+		return "", errors.WithMessage(err, "failed to decode incoming request")
+	}
+
+	// handle number of returned jira users
+	if len(data) == 0 {
+		errorMsg := fmt.Sprintf("We couldn't find the assignee. Please use a Jira member and try again.")
+		return "", fmt.Errorf(errorMsg)
+	}
+
+	if len(data) > 1 {
+		errorMsg := fmt.Sprintf("Your <assignee>, `%s`, matches %d users.  Please make your user request unique.", assignee, len(data))
+		return "", fmt.Errorf(errorMsg)
+	}
+
+	// User is array of one object
+	User := data[0]
```

Comment:
style: local variables are not to start with a Capital letter. suggest `jiraUser`

Reply:
jiraUser is already defined and occupied earlier in the function.  Will how about `user` with lower case?